README.md

```diff
@@ -1,2 +1,54 @@
-# internal-tools-privacy-policy
-Publicly accessible privacy policies for all internal tooling
+# Internal Tools Privacy Policy
+
+Publicly accessible privacy policies for all internal tooling.
+
+## Table of Contents
+
+- [1. Chrome Extension Privacy Policy](#1-chrome-extension-privacy-policy)
+  - [1.1 Information We Collect](#11-information-we-collect)
+    - [1.1.a Authentication Data](#11a-authentication-data)
+    - [1.1.b Access Logs](#11b-access-logs)
+    - [1.1.c Browser Hostnames](#11c-browser-hostnames)
+  - [1.2 Use of Information](#12-use-of-information)
+  - [1.3 Information Sharing and Disclosure](#13-information-sharing-and-disclosure)
+  - [1.4 Data Protection](#14-data-protection)
+  - [1.5 Changes to This Privacy Policy](#15-changes-to-this-privacy-policy)
+  - [1.6 Contact Us](#16-contact-us)
+
+## 1. Chrome Extension Privacy Policy
+
+***This includes Plume Care Extension, Plume Care Extension [beta] and Plume Care Extension [alpha]***
+
+**Effective Date:** Fri Mar 15 2024
+
+This Privacy Policy applies to the Chrome extension developed for internal use by Plume ("we", "us", or "our"). The extension is designed exclusively for employees of Plume ("you" or "your") and is not available to the general public. This policy describes how we collect, use, and protect your information in connection with the extension and is intended to comply with the Health Insurance Portability and Accountability Act (HIPAA) and Google Chrome Store's requirements.
```

Comment:
You should broaden the usage from "employees of Plume" to "all personnel including employees, consultants, contractors of both Plume and Plume Health Medical Group"